src/client/components/pages/MultiYearPlan.tsx

```diff
@@ -85,19 +181,22 @@ const MultiYearPlan: FunctionComponent = (): ReactElement => {
   */
 
   const courseInstance = (course: MultiYearPlanResponseDTO): TableCell[] => (
-    course.semesters.map((semester): TableCell => (
-      <TableCell verticalAlignment={VALIGN.TOP} key={semester.id}>
-        <TableCellList>
-          {semester.instance.faculty.length > 0
-            ? semester.instance.faculty.map((faculty): TableCellListItem => (
-              <TableCellListItem key={faculty.id}>
-                {faculty.displayName}
-              </TableCellListItem>
-            ))
-            : null}
-        </TableCellList>
-      </TableCell>
-    ))
+    range(NUM_SEMESTERS).map((index: number) => {
+      const semester = course.semesters[index];
+      return (
+        <TableCell verticalAlignment={VALIGN.TOP} key={index}>
```

Comment:
It's generally not advised to use array indexes as keys -- what about using the faculty `id` field with the `term` and `academicYear` appended to it?

Reply:
I thought it was ok to use the index as the key since the semester columns would be static? I realized though that I could use the `getSemestersFromYear` function instead of constructing the semesters in the original implementation, so I am now using the semester key, which is `academicYear-term.` (commit 75b6b85).